packages/ckeditor5-basic-styles/docs/features/basic-styles.md

```diff
@@ -15,6 +15,19 @@ The {@link api/basic-styles basic styles} package provides text formatting featu
 
 {@snippet features/basic-styles}
 
+## Related features
+
+Check out also these CKEditor 5 features to gain better control over your content style and format:
+* {@link features/font Font styles} &ndash; Easily and efficiently control the font {@link features/font#configuring-the-font-family-feature family}, {@link features/font#configuring-the-font-size-feature size}, {@link features/font#configuring-the-font-color-and-font-background-color-features text or background color}.
+* {@link features/text-alignment Text alignment} &ndash; Because it does matter whether the content is left, right, centered or justified.
+* {@link features/code-blocks Code block feature}  &ndash; Allows for insertion of various code listings, expanding the 'code' style greatly.
+* {@link features/remove-format Remove format} &ndash; Easily clean basic text formatting.
+
+And be sure to get familiar with these, too:
```

Comment:
I think we could get rid of the second section here and just move "highlight" to the section above. In this way you could have just the "styling/formatting" features here - it's hard to say how document title is related to basic styles at all...